web3/_utils/transactions.py

```diff
@@ -65,7 +65,7 @@ def fill_transaction_defaults(web3, transaction):
     return merge(defaults, transaction)
 
 
-def wait_for_transaction_receipt(web3, txn_hash, timeout=120, poll_latency=0.1):
```

Comment:
@pipermerriam is there any reason to keep the defaults here? The only place this method is used is in `eth.waitForTransactionReceipt`